.circleci/config.yml

```diff
@@ -777,11 +777,12 @@ jobs:
       - attach_workspace:
           at: ~/project
       - quay_login
-      - run: docker run --net host -v /var/run/docker.sock:/var/run/docker.sock -e TEST_RESULTS=~/project quay.io/influxdb/grace:latest
+      - run: mkdir -p ~/project/results
+      - run: docker run --net host -v /var/run/docker.sock:/var/run/docker.sock -v ~/project/results:/grace/test-results/grace-results -e TEST_RESULTS=~/project/results quay.io/influxdb/grace:latest
```

Comment:
What does `TEST_RESULTS` do? I would have thought it would need to be the path mounted inside the docker container

Reply:
That was my first thought as well, but it looks like it's instead used to set the source path for volumes created by `docker compose` within the image: https://github.com/influxdata/grace/search?q=TEST_RESULTS

I'm honestly not 100% sure why it works. Maybe because the `docker compose` CLI is talking to the daemon on the host, so when it passes `source` paths the paths have to be relative to the host and not the intermediary container?

Reply:
ok - well, it works, and it's only build harness, so I'll approve. A mystery to solve another day.